saq/log.py

```diff
@@ -0,0 +1,34 @@
+import logging
+
+
+def generate_logger_config(verbosity):
+    if verbosity == 0:
+        log_level = logging.ERROR
+    elif verbosity == 1:
+        log_level = logging.INFO
+    else:
+        log_level = logging.DEBUG
+
+    return {
+        "version": 1,
```

Comment:
why is there so much stuff here? where did you get this from? is it standard?

Reply:
That's a pretty standard dictConfig. It defines a formatter, which is attached to a handler with a custom log_level, and this handler is attached to the top-level logger "saq".
I decided to put the format explicitly in case you wanted to modify it later on.

Reply:
Wouldn’t it make more sense to pass a logging dict/config as input?

Reply:
@JonasKs Why ? This function exists only to generate the right dictConfig based on the verbosity level the user choose via the cli.

Reply:
Oups sorry, actually the `datefmt` attribute is not needed, my bad.